gtsam/hybrid/HybridGaussianConditional.h

```diff
@@ -207,6 +182,23 @@ class GTSAM_EXPORT HybridGaussianConditional
   /// @}
 
  private:
+  /// Helper struct for private constructor.
+  struct ConstructorHelper;
```

Comment:
Do you think it makes sense to put this in its own file (or in HybridGaussianFactor.h`) so we can reuse it? Perhaps not since we record the `minNegLogConstant` but the semantics should be the same as using the value in HybridGaussianFactor.

Reply:
Probably not. Hiding will change things anyway.